install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -57,12 +57,14 @@ global:
 
     # Sets the destination Statsd in envoy (the value of the "--statsdUdpAddress" proxy argument
     # would be <host>:<port>).
-    # Can also be disabled (e.g. when Mixer is not installed).
-    # For remote clusters the host should be the statsd prom bridge in the primary control plane cluster.
+    # Disabled by default.
+    # The istio-statsd-prom-bridge is deprecated and should not be used moving forward.
     envoyStatsd:
+      # If enabled is set to true, host and port must also be provided. Istio no longer provides a statsd collector.
       enabled: false
```

Comment:
should we even remove it from the values.yaml?  what is the value to have it as a config param since users should not be used.